website/content/docs/agent/autoauth/index.mdx

```diff
@@ -221,5 +259,87 @@ auto_auth {
       path = "/tmp/file-foo"
     }
   }
+
+  sink {
+    type = "file"
+
+    config = {
+      path = "/tmp/file-bar"
+    }
+  }
 }
 ```
+
+~> The [corresponding JSON format](#json-sinks) _must_ specify a `"sinks" :
+[...]` array to encapsulate all `sink` JSON objects.
+
+#### JSON sinks
```

Comment:
Would it be better to rephrase this to `sinks in JSON format`? JSON sink may be a confusing title.